### PR TITLE
Expose log files

### PR DESCRIPTION
**Pull request by zachmullen**
A common pain point during deployment is that log files aren't exposed via the web app, meaning admins have to SSH into the machine, find the log file, and tail it. This exposes the same functionality via an easy-to-use REST endpoint that also shows the path to the log file it's reading from.

@mgrauer we discussed this earlier today

### CODE REVIEW

File: girder/api/v1/system.py [hunk 1]
```diff
@@ -373,3 +376,37 @@ def systemConsistencyCheck(self, params):
...
+        def stream():
+            yield '=== Last %d bytes of %s: ===\n\n' % (
+                min(length, filesize), path
+            )
+
+            with open(path, 'rb') as f:
+                if length < filesize:
+                    f.seek(-length, os.SEEK_END)
+                while True:
+                    data = f.read(65536)
```
Comment (cpatrick) [inline girder/api/v1/system.py]: image

**Comment by cpatrick** [comment]
I like this. Thanks for not logging to the database. :smile_cat:

**Comment by zachmullen** [comment]
@cpatrick Do you want me to move that magic number to a const variable?

**Comment by cpatrick** [comment]
that works.

**Comment by zachmullen** [comment]
@cpatrick PTAL

**Comment by cpatrick** [comment]
That  extra `FER` just took up too much space. :smile_cat: 

Other than that sub-nit, LGTM. :ship:

**Comment by zachmullen** [comment]
I could've used C naming conventions and called it `__LB_LN_` or something ;)

**Pull request merged**